src/planwise/engine/raster.clj

```diff
@@ -58,10 +58,10 @@
 
 (defn- with-open-raster
   [path f]
-  (if-let [dataset (gdal/Open path gdalconst/GA_ReadOnly)]
+  (if-let [provider-set (gdal/Open path gdalconst/GA_ReadOnly)]
```

Comment:
The `dataset` in this file refers to a GDAL dataset and should not be renamed at all.